rpm/tarantool-http.spec

```diff
@@ -20,14 +20,19 @@ This package provides a HTTP server for Tarantool.
 %setup -q -n %{name}-%{version}
 
 %build
-%cmake . -DCMAKE_BUILD_TYPE=RelWithDebInfo
-make %{?_smp_mflags}
-
-%check
-make %{?_smp_mflags} luatest
```

Comment:
Nit: This change is not described/motivated anyhow in the commit message.

Reply:
done